views/quote.py

```diff
@@ -14,6 +14,6 @@ def get_quote(cls, theme, author):
         if error_message:
             return json.dumps({"error_message": error_message}), status
         if len(response) == 0:
-            return json.dumps({"error_message": "No quotes found"}), 501
+            return json.dumps({"error_message": "No quotes found"}), 204
```

Comment:
If it's an empty response, is it really an error? can't we just reply with an empty response? like {"response": None}

Reply:
@josephkim583